pyproject.toml

```
@@ -12,6 +12,8 @@ dependencies = [
   "aiohttp>=3",
   "tzdata>=2024.2 ; platform_system == 'Windows'",
   "mashumaro>=3.14",
+  "sphinx-autodoc-typehints[docs]>=1.23.0",
+  "sphinx[docs]==7.4.7",
```

```suggestion
  "sphinx[docs]>=7.4.7",
```
But I don't think we want to add these as required dependencies.